test/utils/goHelpers/cmd/receiptMMPTProof/main.go

```
@@ -0,0 +1,71 @@
+	logIdx := uint64(1)
+
+	rpcUrl := os.Getenv("OP_SEPOLIA_RPC_URL")
+	alchemyRpc, err := ethclient.Dial(rpcUrl)
+	if err != nil {
+		fmt.Println("error dialing alchemyRpc: ", err)
+		return
+	}
+
+	client := ethclient.NewClient(alchemyRpc.Client())
+
+	ctxWithTimeout, _ := context.WithTimeout(context.Background(), 10000*time.Millisecond)
```

golangci-lint (1.61.0)

lostcancel: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak

(govet)